src/main/java/world/bentobox/aoneblock/oneblocks/OneBlockPhase.java

```diff
@@ -158,7 +166,8 @@ public OneBlockObject getNextBlock(AOneBlock addon, int blockNumber) {
             return getResult(this.getFixedBlocks().get(blockNumber));
         }
         OneBlockObject block = getRandomBlock(probMap, total);
-        if (block.isEntity()) return block;
+        if (block.isEntity())
+            return block;
```

Comment:
Let's put brackets around this return. I know it wasn't in the original, but it could be confusing otherwise.